experiments/ClimaEarth/components/ocean/slab_ocean.jl

```diff
@@ -137,6 +139,12 @@ end
 function Interfacer.update_field!(sim::SlabOceanSimulation, ::Val{:turbulent_energy_flux}, field)
     parent(sim.integrator.p.F_turb_energy) .= parent(field)
 end
+function Interfacer.update_field!(sim::SlabOceanSimulation, ::Val{:surface_direct_albedo}, field::CC.Fields.Field)
```

Comment:
we still don't require surface model simulations to extend `update_field!` for these two albedo fields, right? Just wondering if we should update the docs/explanation for FluxCalculator

Reply:
This is only required if the component in question has time varying albedo calculated outside of the model (e.g. the ocean albedo calculated in the coupler using atmos state). But it's a good idea to clarify this in the docs.